audb/core/load.py

```diff
@@ -899,10 +899,28 @@ def load(
         mixdown: apply mono mix-down
         sampling_rate: sampling rate in Hz, one of
             ``8000``, ``16000``, ``22500``, ``44100``, ``48000``
-        tables: include only tables matching the regular expression or
-            provided in the list
-        media: include only media matching the regular expression or
-            provided in the list
+        attachments: load only attachment files
+            for the attachments
+            matching the regular expression
+            or provided in the list.
+            Use ``[]`` to not load any attachment files
```

Comment:
For consistency with `tables` and `media` maybe change to:

```suggestion
            If set to ``[]`` no attachments are loaded
```